nimble/host/mesh/src/prov.c

```diff
@@ -177,7 +179,7 @@ static struct os_mbuf *rx_buf;
 #define PROV_BUF(len) NET_BUF_SIMPLE(PROV_BUF_HEADROOM + len)
 
 static struct prov_link link;
-
+static struct os_callout protocol_timer;
```

Comment:
you cannot use `os_callout` in nimble code - use `ble_npl_callout` instead

Reply:
Done

Reply:
Done